projects/addon-doc/components/navigation/navigation.component.ts

```diff
@@ -169,9 +170,9 @@ export class TuiDocNavigation {
     protected onFocusSearch(event: KeyboardEvent): void {
         if (
             event.code === 'Slash' &&
```

Comment:
`(window:keydown)` could be changed to `(window:keydown./)` to avoid checking for key code here.

Reply:
Yes, I tried that initially, but it wouldn't work for other languages